content/articles/data-encryption-with-qr-codes-in-android/index.md

```diff
@@ -0,0 +1,471 @@
+### Introduction
+Quick response (QR) codes are a type of barcode, or scannable series, that can store a range of data, including web links, credit card info, contact information, and even freebies. QR codes are used to encode information in a format that only machines, not humans, can understand.
+
+In this tutorial, we will learn how to generate QR codes to store simple text format data in an Android application.
+
+#### Why use Quick Response Codes?
+QR codes have the following advantages:
+1. Unlike other barcodes, they are not subject to the same security risks as traditional barcodes.
+2. QR codes hold large amounts of data -over 2500 characters.
+3. They minimize processing errors since they are difficult to tamper with.
+4. Unlike the traditional barcode, QR codes stores data in 2D (both vertically and horizontally).
+
+### Prerequisites
+To follow along with this tutorial, you'll need to be conversant with:
+- Creating Android apps with the Kotlin programming language.
+- Imperative paradigm in Android (XML).
+- ViewBinding.
+- Android permissions.

```

Comment:
Hello @MaurineM, please provide the necessary links for these tools and libraries.